src/styles/pages/demo.scss

```diff
@@ -80,6 +80,12 @@
     &:focus {
       outline: none;
     }
+    &:disabled {
```

Comment:
To have the text be centered as in the designs, let's use flexbox for the button and absolute positioning for the spinner

```sass
  button.checkout {
    margin-top: 18px;
    margin-bottom: 18px;
    background: #fade60;
    border: none;
    color: hsl(0, 0%, 0%);
    width: 298px;
    height: 50px;
    border-radius: 5px;
    text-align: center;
    font-weight: bold;
    font-size: 18px;

    position: relative;
    display: flex;
    align-items: center;
    justify-content: center;

    &:hover,
    &:active,
    &:focus {
      outline: none;
    }
    &:disabled {
      background: #bababa;
    }
    > .spinner {
      position: absolute;
      left: 16px;
    }
  }
```